tests/test_kv_app_multi_workers.cc

```diff
@@ -9,10 +9,11 @@ void StartServer() {
 }
 
 void RunWorker(int customer_id) {
-  if (!IsWorker()) return;
   Start(customer_id);
+  if (!IsWorker()) {
+    return;
+  }
```

Comment:
why moving this line behind?

Reply:
the major reason is that this is_worker relies on the set of role variable 

https://github.com/dmlc/ps-lite/blob/aee325276bccb092f516df0bce30d3a8333f4038/src/postoffice.cc#L20-L24

if we put it before, it will always return false,